glove/glove.py

```diff
@@ -227,6 +227,37 @@ def most_similar(self, word, number=5):
 
         return self._similarity_query(self.word_vectors[word_idx], number)[1:]
 
+    def _similarity(self, word1_vec, word2_vec):
+        dst = (np.dot(word1_vec, word2_vec)
+               / np.linalg.norm(word1_vec)
+               / np.linalg.norm(word2_vec))
+
+        return dst
+
+    def similarity(self, word1, word2):
+        """
+        Return the similarity measure between word1 and word2.
+        """
+
+        if self.word_vectors is None:
+            raise Exception('Model must be fit before querying')
+
+        if self.dictionary is None:
+            raise Exception('No word dictionary supplied')
+        
+        try:
+            word1_idx = self.dictionary[word1]
+        except KeyError:
+            raise Exception('Word not in dictionary')
+
+        try:
+            word2_idx = self.dictionary[word2]
+        except KeyError:
+            raise Exception('Word not in dictionary')
+
+        return self._distance(self.word_vectors[word1_idx],
```

Comment:
Does this actually work? Maybe a leftover from the previous version?

Reply:
@owo  _distance is not defined in the current version anyway. I tried running the code with the added functions _similarity and similarity but ended up with AttributeError: 'Glove' object has no attribute '_distance'